### PR TITLE
Updates to WPF documentation

### PR DESCRIPTION
**Pull request by luispandolfi**
Updated WPF documentation to be similar in structure to the Web documentation.

### CODE REVIEW

**Review by brianlagunas — CHANGES_REQUESTED** [review]
Please make these same changes in the Japanese docs. `i18n/ja/docusaurus-plugin-content-docs/current/wpf`

Just copy and paste the Japanese text.

File: i18n/ja/docusaurus-plugin-content-docs/current.json [hunk 1]
```diff
@@ -314,5 +314,21 @@
   "sidebar.webSidebar.doc.Configure Export": {
     "message": "エクスポートの構成",
     "description": "The label for the doc item Configure Export in sidebar webSidebar, linking to the doc web/configure-export"
+  },
```
Comment (brianlagunas) [inline i18n/ja/docusaurus-plugin-content-docs/current.json:317]: How did you generate these? Did you run the command line or did you do it manually?

**Comment by luispandolfi** [comment]
> Please make these same changes in the Japanese docs. `i18n/ja/docusaurus-plugin-content-docs/current/wpf`
> 
> Just copy and paste the Japanese text.

Did it

File: i18n/ja/docusaurus-plugin-content-docs/current.json (hunk 1 above)
Comment (brianlagunas) [reply i18n/ja/docusaurus-plugin-content-docs/current.json:317]: Can you run the translation string command for English too? When you generate the translation strings, but you must generate for both English and Japanese.

**Review by brianlagunas — APPROVED** [review]

**Pull request merged**